### PR TITLE
DOC-970: Add known issues for 5.9

### PR DESCRIPTION
**Pull request by JamesToohey**
Related Ticket: DOC-970

Description of Changes:
Known issues for TinyMCE

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] ~`_data/nav.yml` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed
- [ ] Product Manager has reviewed

### CODE REVIEW

**Comment by lnewson** [comment]
@JamesToohey the branch name/related Jira doesn't appear to be correct here? DOC-970 is the RTC known issues, this one should be DOC-866.

**Review by tylerkelly13 — APPROVED** [review]

File: release-notes/release-notes59.md [hunk 1]
```diff
@@ -417,13 +417,26 @@ This section describes issues that users of {{site.productname}} 5.9 may encount
...
+- [The new table color map settings do not apply to color pickers in the table dialogs](#knownissue1)
+- [Issues with the new _Table Column Header_ and _Table Row Header_ toolbar buttons and menu items](#knownissue2)
 
-###
+<a class="anchor" id="knownissue1"></a>
+### The new table color map settings do not apply to color pickers in the table dialogs
 
-**Issue**: This issue affects ....
+Issue
+: This issue affects users of the new `table_background_color_map` and `table_border_color_map` options. These will change the set of colors used in their respective toolbar buttons and menu items, but not the table dialog.
```
Comment (lnewson) [inline release-notes/release-notes59.md:427]: This doesn't impact users, it impacts integrators.

**Review by lnewson — CHANGES_REQUESTED** [review]
Requesting changes because of the incorrect Jira numbers used.

File: release-notes/release-notes59.md (hunk 1 above)
Comment (tylerkelly13) [reply release-notes/release-notes59.md:427]:
```suggestion
: This issue affects the new `table_background_color_map` and `table_border_color_map` options. Setting these options will change the set of colors used in their respective toolbar buttons and menu items, but not the table dialog.
```

**Pull request closed without merging**